client/src/components/Admin/OrganizationEdit/ContactDetails.js

```diff
@@ -79,7 +79,11 @@ export default function ContactDetails({
         </Grid>
         <Grid item sm={6} xs={12}>
           <div>
-            <Label id="pinterest" label="Pinterest" />
+            <Label
+              id="pinterest"
+              label="Pinterest"
+              tooltipTitle="URL must start with 'https://pinterest.com/'"
```

Comment:
in the last stand up meeting, we've talked about adding a clickable link for the user so they can click it and test that the link works. Maybe we can add it next to the tooltip. It can just be a `<Button variant="text" size="small">` 👌

Reply:
since we're already adding the url, the tooltip should probably just say that it should be a valid username. Here's what's on pinterest [help center](https://help.pinterest.com/en/article/edit-your-profile#:~:text=For%20example%2C%20Pinterest%20has%20the,between%20three%20and%2030%20characters)

Reply:
I think we can update the tooltip to something like `Must be a valid handle without the url`